extensions/adobe/experience/primetime-authentication/application.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":   "https://ns.adobe.com/experience/primetime-authentication/application",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Application extensions for Adobe Primetime Authentication",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/application"],
```

Comment:
Lets see if we can actually add these to the standard Application schema.

Reply:
This would be great.
An application has 3 "zoom levels" in our system, as we consider the following hierarchy:
- The actual commercial name, as advertised by the owner (e.g. WatchESPN)
- The software package that gets shipped to a particular marketplace (e.g. WatchESPN iOS version 3.11)
- A particular installation of such a software package on some device (e.g. the WatchESPN instance on my very phone).